Source/WebKit/WebProcess/GPU/graphics/RemoteImageBufferProxy.cpp

```diff
@@ -205,13 +210,17 @@ void RemoteImageBufferProxy::didCreateBackend(std::optional<ImageBufferBackendHa
     // This should match RemoteImageBufferProxy::create<>() call site and RemoteImageBuffer::create<>() call site.
     // FIXME: this will be removed and backend be constructed in the contructor.
     std::unique_ptr<ImageBufferBackend> backend;
+#if HAVE(IOSURFACE)
     if (renderingMode() == RenderingMode::Accelerated) {
         if (canMapBackingStore())
-            backend = AcceleratedImageBufferShareableMappedBackend::create(parameters(), WTFMove(*handle));
+            backend = ImageBufferShareableMappedIOSurfaceBackend::create(parameters(), WTFMove(*handle));
         else
-            backend = AcceleratedImageBufferRemoteBackend::create(parameters(), WTFMove(*handle));
+            backend = ImageBufferRemoteIOSurfaceBackend::create(parameters(), WTFMove(*handle));
     } else
-        backend = UnacceleratedImageBufferShareableBackend::create(parameters(), WTFMove(*handle));
+        backend = ImageBufferShareableBitmapBackend::create(parameters(), WTFMove(*handle));
+#else
+    backend = ImageBufferShareableBitmapBackend::create(parameters(), WTFMove(*handle));
+#endif
```

Comment:
This code can be shortened 

```
    if (renderingMode() == RenderingMode::Accelerated) {
#if HAVE(IOSURFACE)
        if (canMapBackingStore())
            backend = ImageBufferShareableMappedIOSurfaceBackend::create(parameters(), WTFMove(*handle));
        else
            backend = ImageBufferRemoteIOSurfaceBackend::create(parameters(), WTFMove(*handle));
#endif 
    } else
        backend = ImageBufferShareableBitmapBackend::create(parameters(), WTFMove(*handle));
```

Cairo ports should not be setting the renderingMode to RenderingMode::Accelerated.